docs_user/modules/con_comparing-configuration-files-between-deployments.adoc

```diff
@@ -36,38 +39,33 @@ ocp_local_config_path=/tmp/ocp
 connection=local
 ssh_cmd=""
 ----
-
-Os-diff uses `ssh_cmd` to access your {OpenStackPreviousInstaller} host via SSH,
-or the host where your cloud is accessible and the podman/docker binary is installed
-and allowed to interact with the running containers. This option could have a different form:
-
++
+<1> Instructs os-diff to access your {OpenStackPreviousInstaller} host through SSH.
+<2> The host to use to access your cloud, and the podman/docker binary is installed and allowed to interact with the running containers. You can leave this key blank.
++
 ----
 ssh_cmd=ssh -F ssh.config standalone
 director_host=
 ----
-
-----
-ssh_cmd=ssh -F ssh.config
-director_host=standalone
-----
-
++
 or without an SSH config file:
```

Comment:
Regarding the “without an SSH config file” examples,  Why is `my.undercloud.local` appended to `-i /home/user/.ssh/id_rsa stack@` in the first example, but then split up in the second example?
```
ssh_cmd=ssh -i /home/user/.ssh/id_rsa stack@
director_host=my.undercloud.local
```

Reply:
Both writting can be valid, but I think it might be confusing.
I'm wondering if we might remove the 2nd example